boards/phytec/mimx93_phyboard_nash/mimx93_phyboard_nash_a55_defconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2022 NXP
+#
```

Comment:
Add PHYTEC here.

Reply:
I changed the copyright notice to PHYTEC, as those are just some board related configuration flags.